app/views/pages/index.html.erb

```diff
@@ -1,16 +1,7 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-    
-</head>
-
 <body>
```

Comment:
your layout already has `<body></body>` tags that your view is yielded into, so you don't need them here